xtask/src/main.rs

```diff
@@ -32,6 +36,9 @@ enum Commands {
         /// Actually build projects, instead of just checking them
         #[arg(short, long)]
         build: bool,
+        /// Just print what would be tested
+        #[arg(short, long)]
+        dry: bool,
```

Comment:
While doing some research for this same issue, I found out that we have an `options` subcommand that does mostly the same, was this on purpose?

Reply:
true it's definitely the same - personally, I would prefer `dry_run` and remove the `options` command (and rename `dry` to `dry_run`)

What do you think?

Reply:
Sounds good to me!